### ISSUE TITLE
Feature: enable typing path when creating or importing

### ISSUE THREAD

**Issue by mslourens** [nrwl/angular-console#206]
When creating or importing a project, you have to select the directory of the project. Currently you can only select the directory by clicking through the tree. Especially in Windows this takes too much time (the Documents directory in which my projects are stored is too deep for my liking). It would be much more convenient to be able to enter a path to a directory and add a browse button for those who still want to be able to select to folder from a tree.

In short: change the current path selecting behavior to an input field with a browse button.

**Comment by mrmeku** [comment]
Tracking in #237
